index.html

```diff
@@ -13680,32 +13680,18 @@ <h2>Interface Mixin <dfn>ARIAMixin</dfn></h2>
 				[CEReactions] attribute DOMString? ariaValueNow;
 				[CEReactions] attribute DOMString? ariaValueText;
 			};
+			Element includes ARIAMixin;
 		</pre>
 
-		<p>Interfaces that include <code>ARIAMixin</code> must provide the following algorithms:</p>
-
-		<ul>
-			<li><dfn><code>ARIAMixin</code> getter steps</dfn>, which take the host interface instance, IDL attribute name, and content attribute name, and must return a string value; and
-			<li><dfn><code>ARIAMixin</code> setter steps</dfn>, which take the host interface instance, IDL attribute name, content attribute name, and string value, and must return nothing.</li>
-		</ul>
-
-		<p>For every IDL attribute <var>idlAttribute</var> defined in <code>ARIAMixin</code>, on getting, it must perform the following steps:</p>
+		<p>For every IDL attribute <var>idlAttribute</var> defined in <code>ARIAMixin</code> when included on an element:</p>
 
 		<ol>
-			<li><p>Let <var>contentAttribute</var> be the ARIA content attribute determined by looking up <var>idlAttribute</var> in the ARIA Attribute Correspondence table.</p></li>
+			<li><p>Let <var>contentAttributeName</var> be the local name of the ARIA content attribute determined by looking up <var>idlAttribute</var> in the ARIA Attribute Correspondence table.</p></li>
 
-			<li><p>Return the result of running the <a><code>ARIAMixin</code> getter steps</a>, given this, <var>idlAttribute</var>, and <var>contentAttribute</var>.</p></li>
+			<li><p><var>idlAttribute</var> must <a>reflect</a> <var>contentAttributeName</var> and <a>support <code>ElementInternals</code></a>.</p></li>
```

Comment:
Looks like `reflect` and `support ElementInternals` references are missing.

Reply:
```suggestion
			<li><p><var>idlAttribute</var> must <a href="https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflect">reflect</a> <var>contentAttributeName</var> and <a href="https://html.spec.whatwg.org/multipage/custom-elements.html#the-elementinternals-interface">support <code>ElementInternals</code></a>.</p></li>
```

Reply:
@pkra move these refs to the common files if needed.

Reply:
@cookiecrook @keithamus this is just a regular exported term, see https://github.com/w3c/aria#references-to-other-w3c-specs for guidance but in short `[=reflect=]` and `{{ElementInternals}}` should work (there are other references in this style throughout the spec, too).

Reply:
```suggestion
			<li><p><var>idlAttribute</var> must [=reflect=] <var>contentAttributeName</var> and support {{ElementInternals}}.</p></li>
```